package.json

```diff
@@ -43,5 +43,8 @@
   "engines": {
     "node": ">=20.0.0"
   },
-  "packageManager": "yarn@3.5.0"
+  "packageManager": "yarn@3.5.0",
+  "dependencies": {
+    "node-gyp": "^10.1.0"
+  }
```

Comment:
Nit: conventionally this object is placed above `"devDependencies"`. Do you mind moving it?

Reply:
Sure, I made this commit now.